import_file/utils.go

```diff
@@ -0,0 +1,54 @@
+package import_file
+
+import (
+	sp "cloud.google.com/go/spanner"
+	"context"
+	"fmt"
+	spannerclient "github.com/GoogleCloudPlatform/spanner-migration-tool/accessors/clients/spanner/client"
+	"github.com/GoogleCloudPlatform/spanner-migration-tool/internal"
+	"github.com/GoogleCloudPlatform/spanner-migration-tool/logger"
+	"github.com/GoogleCloudPlatform/spanner-migration-tool/spanner/writer"
+	"os"
+	"sync/atomic"
+)
+
```

Comment:
Can we avoid creating a `utils.go` file? In my experience, it becomes a dumping ground for all sorts of random, unrelated methods? 

For example, can we definge the `getBatchWriterWithConfig` method in the `writer.BatchWriter` struct itself? Do you see a risk?

Reply:
Have moved it to writer.BatchWriter